include/ipc.h

```diff
@@ -14,6 +14,7 @@ enum ipc_command_type {
 	IPC_GET_VERSION = 7,
 	IPC_GET_INPUTS = 8,
 	IPC_GET_CLIPBOARD = 9,
+	IPC_GET_SEATS = 10,
```

Comment:
It looks like i3 has added some new messages.

https://github.com/i3/i3/blob/next/include/i3/ipc.h

Why don't we bump our extensions up to 100+.